elb_log_ingestor/elb_log_parse.py

```diff
@@ -324,24 +367,33 @@ def remove_empty_fields(d: typing.Dict) -> typing.Dict:
     return d
 
 
-def generate_id(entry: typing.Dict) -> str:
+def generate_id(entry: typing.Dict, line: str) -> str:
     """
     Generate a fairly unique key for a log entry
     """
-    if "@alb" in entry:
-        # ALBs already have one
-        key = entry["@alb"]["trace_id"]
-    else:
-        # for ELBs, take the elb id, client socket, timestamp, and size of the client request`
-        key = ":".join(
-            [
-                entry["@elb"]["elb"]["id"],
-                entry["@elb"]["client"]["ip"],
-                str(entry["@elb"]["client"]["port"]),
-                entry["@timestamp"],
-                str(entry["@elb"]["received_bytes"]),
-            ]
-        )
+    key = line
```

Comment:
we were flagging a _lot_ of false duplicates before. Using the line as the key fixed that.

Why did I leave the `entry` parameter? I don't know.